src/__tests__/logger.test.js

```diff
@@ -104,21 +104,197 @@ describe('logger', function () {
     expect(logger.outputEnabled).toBe(false);
   });
 
-  it('logs deprecations to the console when logger.outputEnabled=true', function () {
-    var message = '_satellite.deprecatedFeature is officially deprecated';
-    logger.outputEnabled = true;
-    logger.deprecation(message);
+  describe('logger.deprecation', function () {
```

Comment:
This was the original ask of the ticket `pdcl-13979`: Turbine was too catty in a production environment because `logger.deprecation` was allowed to call the console even when `setDebug(false)` was called.